Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -392,7 +392,7 @@ public async Task RenderActionsAfterAppendComplete()
             var stateStore = new TrieStateStore(new MemoryKeyValueStore());
             IActionRenderer<DumbAction> renderer = new AnonymousActionRenderer<DumbAction>
             {
-                ActionRenderer = (_, __, nextStates) =>
+                ActionRenderer = (action_, ctx, nextStates, logs) =>
```

Comment:
```suggestion
                ActionRenderer = (action, ctx, nextStates, logs) =>
```

Or only adding `logs` parameter:

```suggestion
                ActionRenderer = (_, __, nextStates, logs) =>
```